README.md

```diff
@@ -448,6 +448,35 @@ Similarly to the example above, if your Markdown includes Mermaid graph specific
 
 For more details and configuration options, see the [Mermaid usage docs](https://mermaid-js.github.io/mermaid/#/usage).
 
+## Changing documentation over time
+
+As your project grows, your documentation may very likely change, even structurally. There are a few important things to consider in this regard:
+
+- Links to your *extras* will break if you change or move file names.
+- Links to your *modules, and mix tasks* will change if you change their name.
+- Links to *functions* are actually links to modules with anchor links. If you change the function name, the link does
+  not break but will leave users at the top of the module's documentation.
+
+Because these docs are static files, when a user gets to a page that is not found, they will see a generic 404 page.
```

Comment:
Can we fix this on `hexdocs.pm`? On a not found page for a package that actually exists, could we render a nicer message, or at least a link to the page?

Example not found page: https://hexdocs.pm/ash/something-random

![CleanShot 2024-09-17 at 08 49 56](https://github.com/user-attachments/assets/1f30d387-2379-4c84-982f-fa91465ee8f5)

Reply:
You would have to ask them.